app/views/catalog/citation.html.erb

```diff
@@ -0,0 +1,2 @@
+<p>Please verify the citations below with the appropriate style guide.</p>
```

Comment:
is this overriding the `citation.html.erb` view that comes from blacklight I assume? Or how/where does this `citiation.html.erb` gets rendered?

Reply:
Correct.  This is the file in blacklight https://github.com/projectblacklight/blacklight/blob/v6.10.1/app/views/catalog/citation.html.erb